docker-compose.yml

```diff
@@ -22,9 +22,12 @@ services:
 
   db:
     profiles: ["dev","prod"]
-    image: postgres:16
+    build:
+      context: ./postgres
+      dockerfile: postgres.Dockerfile
```

Comment:
Why don’t we use official image? https://github.com/pgvector/pgvector#docker

Reply:
Probably because I'm retarded 🤟🏻 but running on official image, I get this message when trying to migrate
![image](https://github.com/knmlprz/ChatKNML/assets/32245772/b1b4213a-6331-480b-98b9-5c8b3a16d429)